src/EditorFeatures/CSharpTest/Structure/MetadataAsSource/InvalidIdentifierStructureTests.cs

```diff
@@ -72,6 +72,7 @@ public async Task IdentifierThatLooksLikeCode()
 
         await VerifyBlockSpansAsync(code,
             Region("textspan3", "/* now everything is commented (); ...", autoCollapse: true),
+            Region("textspan2", "hint2", CSharpStructureHelpers.Ellipsis, autoCollapse: false),
```

Comment:
Reset this to its prev state: see https://github.com/dotnet/roslyn/pull/71064/files#diff-0fd1ed24f6843351146536bd47576962e5ff059c6a5834075c2d7beef4b50cbc